src/utils/abi-decoder.ts

```diff
@@ -112,18 +112,21 @@ export default class ABIDecoder {
         // const abiItem = await this.getABIFunctionFromExternalSource(methodID)
         const abiResult = this.methodSigs[methodID] || (await this.db.getFirstABIForHexSignature(methodID))
         const abiItem = abiResult ? ABI_FunctionZ.parse(abiResult) : null
-
         if (abiItem) {
-            const decoded = abiCoder.decodeParameters(abiItem.inputs, data.slice(10))
+            let decoded = {};
+            try {
+                decoded = abiCoder.decodeParameters(abiItem.inputs, data.slice(10))
+            } catch {
+                // TODO: log this error
```

Comment:
I started to get this error a bunch too and I can't figure out why it was working before but now `ethersABICoder.decode` within this function is throwing errors a bunch

Reply:
ok I fixed it in #18